compiler/rustc_session/src/options.rs

```diff
@@ -1425,6 +1425,8 @@ options! {
     flatten_format_args: bool = (false, parse_bool, [TRACKED],
         "flatten nested format_args!() and literals into a simplified format_args!() call \
         (default: no)"),
+    force_full_debuginfo: bool = (true, parse_bool, [TRACKED],
+        "force all codegen-units to have full debuginfo even for -C debuginfo=1. see #64405 (default: yes)"),
```

Comment:
This is misleading. It merely causes more debuginfo to be added, not everything. For example function arguments aren't added to the debuginfo. Just basic things like the linkage name and demangled function name.